Hcal/testbeam04-2022/pedestals/testbeam22_run_ts_events.csv

```diff
@@ -0,0 +1,77 @@
+# Made by hand May 16th, 2022
+# TS_EVENTS are in thousands
```

Comment:
Not sure what to do with this file.

Reply:
Joe asked for a file with TS events separately. We could probably do without it at some point, however.